pkg/config/draftconfig.go

```diff
@@ -3,12 +3,21 @@ package config
 import (
 	"errors"
 	"fmt"
+	"io/fs"
 
 	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
 )
 
+const draftConfigFile = "draft.yaml"
```

Comment:
where is this used? I can't find it other than a test

Reply:
currently only test, will be used in future changes within this file as well